docs/source/community.rst

```diff
@@ -0,0 +1,69 @@
+sits within the broader Python numeric ecosystem commonly referred to as PyData
+or SciPy.
+
+Discussion
+----------
+
+Conversation happens in the following places:
+
+1.  **Usage questions** are directed to `Stack Overflow with the #intake tag`_.
+    Intake developers monitor this tag.
+2.  **Bug reports and feature requests** are managed on the `GitHub issue
+    tracker`_
+3.  **Chat** occurs on at `gitter.im/ContinuumIO/intake
+    <https://gitter.im/ContinuumIO/intake>`_.  Note that
+    because gitter chat is not searchable by future users we discourage usage
+    questions and bug reports on gitter and instead ask people to use Stack
+    Overflow or GitHub.
+4.  **Monthly developer meeting** happens the first Thursday of the month at
+    9:30 US Central Time in `this video meeting <https://zoom.us/j/9925959983>`_.
+    Meeting notes are available at
+    https://docs.google.com/document/d/1GVcqubeTGICQzPGq_BZxn2Bdtf31lnvcrRxMY3GPCPsF/edit
+
+    .. raw:: html
+
+       <iframe src="https://calendar.google.com/calendar/embed?src=intake.dev%40gmail.com" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
+
+    You can subscribe to this calendar to be notified of changes:
+
+    * `Google Calendar <https://calendar.google.com/calendar/embed?src=intake.dev%40gmail.com>`__
+    * `iCal <https://calendar.google.com/calendar/ical/intake.dev%40gmail.com/public/basic.ics>`__
+
+.. _`Stack Overflow with the #intake tag`: https://stackoverflow.com/questions/tagged/intake
+.. _`GitHub issue tracker`: https://github.com/intake/intake/issues/
+
+
+Asking for help
+---------------
+
+We welcome usage questions and bug reports from all users, even those who are
+new to using the project.  There are a few things you can do to improve the
+likelihood of quickly getting a good answer.
+
+1.  **Ask questions in the right place**:  We strongly prefer the use
+    of Stack Overflow or GitHub issues over Gitter chat.  GitHub and
+    Stack Overflow are more easily searchable by future users, and therefore is more
+    efficient for everyone's time.  Gitter chat is strictly reserved for
+    developer and community discussion.
+
+    If you have a general question about how something should work or
+    want best practices then use Stack Overflow.  If you think you have found a
+    bug then use GitHub
+
+2.  **Ask only in one place**: Please restrict yourself to posting your
+    question in only one place (likely Stack Overflow or GitHub) and don't post
+    in both
+
+3.  **Create a minimal example**:  It is ideal to create `minimal, complete,
+    verifiable examples <https://stackoverflow.com/help/mcve>`_.  This
+    significantly reduces the time that answerers spend understanding your
+    situation, resulting in higher quality answers more quickly.
+
+    See also `this blogpost
+    <http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports>`_
```

Comment:
Perhaps enough for us to refer to MCVE above, perhaps a little weird to be so obviously linked to Dask here

Reply:
Yes, I did a copy of all the stuff on the dask page. Please let me know anything that should be deleted.